# WriteString

std_msgs/Header header
string text
